dbt_schema_builder/relation.py

```diff
@@ -254,3 +254,34 @@ def write_sql(self, raw_schema):
                     self.app, view_type, relation_dict, raw_schema, self.redactions
                 )
                 self.write_sql_file(sql_file_path, sql)
+
+    def write_sql_no_pii(self, raw_schema):
+        """
+        Renders the SQL for this relation and writes out.
+        """
+        relation_dict = self.prep_meta_data()
+
+        if self.is_unmanaged:
+            logger.info(
+                "{}.{} is an unmanaged table, skipping SQL generation.".format(
+                    self.app, self.relation
+                )
+            )
+        else:
+            for view_type in ("SAFE"):
```

Comment:
So, I think you may want to check how this runs in a debugger, this evaluates as just a standard string, so this iterator will return each letter, you probably want to either switch to a list `["SAFE"]` or a tuple by adding `,` so `("SAFE",)`. One of those annoying nuances of the python iterator

Reply:
@dnuzz Appreciate it, changed